spyder/plugins/findinfiles/widgets/main_widget.py

```diff
@@ -499,10 +509,7 @@ def _get_options(self):
         if exclude:
             error_msg = regexp_error_msg(exclude)
             if error_msg:
-                exclude_edit = self.exclude_pattern_edit.lineEdit()
-                exclude_edit.setStyleSheet(self.REGEX_INVALID)
-                tooltip = self.REGEX_ERROR + ': ' + str(error_msg)
-                self.exclude_pattern_edit.setToolTip(tooltip)
+                self.show_error(str(error_msg), True)
```

Comment:
```suggestion
                self._show_error(str(error_msg), True)
```

Let's make this method private because it's not called outside this class.